Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/IndexService.java

```diff
@@ -6,13 +6,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
@jdamerow Not sure what's the problem with formatting. Could you please help me understand. When I save the file in eclipse it does format the edited lines automatically for me.